packages/replayio/src/utils/prompt/prompt.ts

```diff
@@ -38,6 +40,7 @@ export async function prompt({
         case "\x03":
           // \x03 is Ctrl+C (aka "End of text")
           // https://donsnotes.com/tech/charsets/ascii.html
+          onExit?.();
           process.exit(0);
```

Comment:
It's kind of gross the `prompt` is listening for this to begin with IMO. I wasn't sure of a better way to _not_ break that functionality. (Although I should probably have used the `exitProcess` util here rather than calling `process.exit` directly.)

Reply:
> It's kind of gross the prompt is listening for this to begin with IMO.

I don't think we can escape this easily because we are in the raw mode here. One could have some script that could just do anything on CTRL+C in this mode - it loses its default behavior altogether.

> Although I should probably have used the exitProcess util here rather than calling process.exit directly.

I thought about using it here and decided against it as the user expects an immediate~ exit. That function waits for LaunchDarkly to exit etc.

Reply:
Okay. I don't feel strongly about what the "right" thing is to do here. I'll defer!